linear_operator/operators/_linear_operator.py

```diff
@@ -6,11 +6,14 @@
 import math
 import numbers
 import warnings
-from abc import ABC, abstractmethod
+from abc import abstractmethod
 from copy import deepcopy
+from types import EllipsisType
```

Comment:
This is python 3.10+ only. We'll need to support py3.8 for some time so this will require a different solution

Reply:
No problem. I have changed this to: `type("...")`